background/test/app.test.js

```diff
@@ -47,16 +47,16 @@ describe('behavior of cloud function', function() {
       project: projectId,
     });
     const res = await db.collection('/translations').get();
-    res.forEach(element => {
-      element.ref.delete();
+    res.forEach(async element => {
+      await element.ref.delete();
     });
     console.log('Firebase translation collection deleted');
   });
 
   it('should get the correct website', async () => {
     const body = await fetch(`${app}/`);
     const res = await body.status;
-    expect(res).to.equal(200);
+    assert.equal(res, 200);
```

Comment:
It's important to use `assert.strictEqual`, kinda always.  `equal` does not do strict type checking, so things like `assert.equal('200', 200)` would still be true.